src/api/lib/backend/remember_location.rb

```diff
@@ -0,0 +1,33 @@
+module Backend
+  module RememberLocation
+    def singleton_method_added(method_name)
```

Comment:
@ChrisBr This slipped in by accident, right?

Reply:
Oh yeah, removing